lib/mix/tasks/talon.gen.theme.ex

```diff
@@ -220,6 +223,41 @@ defmodule Mix.Tasks.Talon.Gen.Theme do
   end
   defp gen_generators(config), do: config
 
+  def gen_dashboard_template(%{dashboard: true} = config) do
+    binding = Kernel.binding() ++
+      [title_txt: gettext("Dashboard"),
```

Comment:
Sounds good about using `dgettext` but in this case, it doesn't appear we're actually using `'title_txt` anywhere... It was carried over from ExAdmin, but I didn't see it used there either. 

We can use dgettext on `welcome_txt` and `add_txt`, but that text is just placeholder information for the Talon developer and would typically deleted, no?